doc/sphinx/changes.rst

```diff
@@ -14,6 +14,93 @@ Version 8.14
 Summary of changes
 ~~~~~~~~~~~~~~~~~~
 
+Coq version 8.14 integrates many usability improvements, as well
+as an important change in the core language. The main changes include:
+
+  - The :ref:`internal representation <814CaseRepresentation>` of `match` has changed
+    to a more space-efficient and cleaner structure, allowing the fix a completeness
+    issue with cumulative inductive types in the type-checker.
+    The internal representation is now closer to the user-level view of `match`,
+    where the argument context of branches and the inductive binders `in` and `as`
+    do not carry type annotations.
+
+  - A :ref:`new <814CoqNative>` `coqnative` binary performs separate native compilation
+    of libraries, starting from a `.vo` file, and is supported by `coq_makefile`.
+
+  - :ref:`Improvements <814TCCanon>` to typeclasses and canonical structure
+    resolution, allowing more terms to be considered as classes or keys.
+
+  - More control over :ref:`notations <814Notations>` declarations and support
+    for primitive types in string and numeral notations.
```

Comment:
Maybe
```suggestion
    for primitive types in string and number notations.
```
since the command is now `Number Notation`?

Reply:
Indeed